Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _review_vendor:
+
+****************************
+Reviewing for @cilium/vendor
+****************************
+
+What is @cilium/vendor?
+=======================
+
+Team `@cilium/vendor <vendor_team_>`_ is a GitHub team of Cilium contributors
+who are responsible for maintaining the good state of Go dependencies for
+Cilium and it's related projects by reviewing Pull Requests (PRs) that update
+files related to dependency declaration:
+
+* `go.mod <go_dot_mod_>`_
+* `go.sum <go_dot_sum_>`_
+* `vendor/ <vendor_slash_>`_
+
+Each time a non-draft PR touching files owned by the team opens, GitHub
```

Comment:
“Touching” could be interpreted literally or emotionally, which can create confusion.
```suggestion
Each time a non-draft PR modifying files owned by the team opens, GitHub
```

Reply:
Good catch, thanks!